<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>freqSetting</class>
 <widget class="QDialog" name="freqSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="LNAslider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
   <property name="pageStep">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="LNAspinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LNA Gain [dB]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>VGA Gain</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="VGAspinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>14</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSlider" name="VGAslider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>14</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="pageStep">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckBantPow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Antenna port power</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBcancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBok">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBapply">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="console">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>231</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CBfilterPath">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RF Filter Path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
